File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -10,20 +10,17 @@ documentation: UG
 
 ## URL to PDF
 
-To convert website URL or local HTML file to PDF document, refer to the following code snippet.
+To convert website URL or local HTML file to PDF document using [Convert](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.HtmlToPdfConverter.html#Syncfusion_HtmlConverter_HtmlToPdfConverter_Convert_System_String_) method, refer to the following code snippet.
```

Comment:
using [Convert]() method in [HtmlToPdfConverter]() class.

Reply:
Modified